src/topology/category/UniformSpace.lean

```diff
@@ -65,6 +65,9 @@ structure CpltSepUniformSpace :=
 
 namespace CpltSepUniformSpace
 
+-- TODO (Scott): I really don't like having to write any of the definitions in the next section.
+-- It should all be provided later by the forgetful functor, but I'm failing to get things working.
+section
 instance : has_coe_to_sort CpltSepUniformSpace :=
 { S := Type u, coe := CpltSepUniformSpace.α }
```

Comment:
Shouldn't it be a local instance?

Reply:
Yes, it should be, but things break later in the file in a way that I don't understand. If you wanted to have a look that would be very helpful.

Reply:
I managed to clean up `UniformSpace` quite a bit. I'm happier with it, but it's still not as clean as one might hope.

Reply:
Fixed.